cmd/cli.go

```diff
@@ -23,5 +25,11 @@ import (
 type CommonCliArgs struct {
 	rcmd.CommonCliArgs
 	rcmd.LoggingCliArgs
-	BaseUrl string `arg:"--base-url, env" help:"The externally accessible URL on which the OAuth service is listening. This is used to construct manual-upload and OAuth URLs"`
+	BaseUrl      string `arg:"--base-url, env" help:"The externally accessible URL on which the OAuth service is listening. This is used to construct manual-upload and OAuth URLs"`
+	DisableHTTP2 bool   `arg:"--disable-http2, env" default:"true" help:"whether to disable access using the HTTP/2 protocol."`
```

Comment:
I was thinking about that but that would mean tying the remote secrets and spi together a little bit more. For example, I think the `ExposeProfiling` was a mistake because it is actually not supported in spi operator. So I'd rather duplicate a little bit of code to not tie the operators together more than save a couple of lines of code.